doc/website/getting-started/installation.md

```diff
@@ -122,6 +125,14 @@ The `CMakeLists.txt` from `iceoryx_meta` can be used to easily develop iceoryx w
     cmake --build build
     ```
 
+    If the compilation fails it is possible that the compilation failure originates
```

Comment:
```suggestion
    If the compilation fails, it is possible that the compilation failure originates
```

Reply:
I'll appreciate if you could check this sentence, too @@ltbj Thanks :pray:

Reply:
@mossmaurice Here the results from https://writer.com/punctuation-checker/

Your sentence:
```
I'll appreciate if you could check this sentence, too.
```

The corrected version:
```
I'd appreciate it if you could check this sentence, too.
```

The comma is correct but you forgot some words :D. But with the other sentence you were right

Reply:
Agreed with the suggestion to add the comma. Thanks! 

FYI I'm not familiar with writer.com, but I will say my suggestions are based off the Chicago Manual of Style (CMoS).